build/webpack.prod.js

```diff
@@ -0,0 +1,29 @@
+const { merge } = require('webpack-merge');
+const common = require('./webpack.base.js').config;
+
+const entries = {
+    'dash.all': './index.js',
+    'dash.mss': './src/mss/index.js',
+    'dash.mediaplayer': './index_mediaplayerOnly.js',
+    'dash.protection': './src/streaming/protection/Protection.js',
+    'dash.reporting': './src/streaming/metrics/MetricsReporting.js'
+}
```

Comment:
In Gruntfile.js there was the ```offline``` target. Why this is dropped here?

Reply:
Thanks, this has been removed by mistake.
Restored